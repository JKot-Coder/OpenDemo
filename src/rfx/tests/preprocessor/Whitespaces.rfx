// :PREPROCESSOR

// Teset correct handling whitespace in define
#define FOO BAR /*
    */   !   		LINE

FOO