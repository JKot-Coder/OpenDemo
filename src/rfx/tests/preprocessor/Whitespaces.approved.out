SourceOutput:[
    #line 7 "../src/rfx/tests/preprocessor/Whitespaces.rfx"
    BAR ! LINE
]
Diagnostic:[
]
